### PR TITLE
Update Activity Time Formatting in New activity provider

### PR DESCRIPTION
**Pull request by michaelwehrley**
Update Activity Time Formatting

* Update Activity dateFormat in new activity provider (activity_tracker.js)
* Add Moment wrapper.
* Replace `moment()` with `momentjs()`.
* Name spaced `momentjs` to global `sc`.

[#91861818]

### CODE REVIEW

File: app/assets/javascripts/think_feel_do_engine.js [hunk 1]
```diff
@@ -28,4 +28,5 @@
 //= require think_feel_do_engine/validate_social_sharing
 //= require think_feel_do_engine/markdown_editor.js
 //= require think_feel_do_engine/media_access_events.js
+//= require think_feel_do_engine/momentWrapper
```
Comment (michaelwehrley) [inline app/assets/javascripts/think_feel_do_engine.js]: @ericcf I wrote this as part of trying to improve the code within `activity_tracker.js`  Let me know your thoughts.  Thanks!

Comment (ericcf) [inline app/assets/javascripts/think_feel_do_engine.js]: One thought - call it a different name. One purpose in "wrapping" a service is to decouple the rest of your code from it. Naming the wrapper after the original service still couples it.

File: app/assets/javascripts/think_feel_do_engine/momentWrapper.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+(function(momentFn) {
+  'use strict';
+
+  window.sc = window.sc || {};
+
+  function momentService(momentjs) {
+    return momentjs;
+  };
+
+  return window.sc.momentjs = momentService(momentFn);
```
Comment (ericcf) [inline app/assets/javascripts/think_feel_do_engine/momentWrapper.js]: Why use a method call as opposed to just `window.sc.momentjs = momentFn;`? Also, why use a `return` statement?

File: app/assets/javascripts/think_feel_do_engine.js (hunk 1 above)
Comment (michaelwehrley) [inline app/assets/javascripts/think_feel_do_engine.js]: Interesting point.  Thanks.

**Pull request closed without merging**